pkg/arvo/sys/hoon.hoon

```diff
@@ -1915,7 +1915,7 @@
 ::
 ++  curr                                                ::  curry right
   |*  [a=$-(^ *) c=*]
-  |*  b=_+<+.a
+  |*  b=_+<-.a
```

Comment:
This should actually be `b=,.+<-.a` -- leaving the existing face on prevents us from finding the `$` when handling higher-ordered gates.

The equivalent change is needed for `+cury` (`c=_,.+<+.a`). Can you fix them both here?

Reply:
ok, stripped faces in curry functions and also in ++corl